### ISSUE TITLE
i believe the camera matrix needs to be inversed to get the rotation correct

### ISSUE THREAD

**Issue by qiangt** [StanislavPetrovV/Software_3D_engine#9]
first of all, thank you for providing this wonderful 3D render tutorial. It helps me to understand the whole process, instead of a black box that you just call it. 

However, I do have a question on the camera_matrix function in camera.py.  When I pressed left/right/up/down, the rotation doesn't look right (it feels more like translation). After I searched a bit, I think the camera_matrix needs to be inversed before the function return it, based on this [blog](https://towardsdatascience.com/camera-extrinsic-matrix-with-example-in-python-cfe80acab8dd)

```
    def camera_matrix(self):
        self.camera_update_axii()
        return np.linalg.inv(self.translate_matrix() @ self.rotate_matrix())
```

After I revert the camera matrix, the rotation does looks way better. Although now the tank looks up side down. 
Would like to know what do you think.

**Comment by willmil11** [comment]
Yeah but at the first stage where theres only a cube the camera_update_axii() doesnt exists so i have to remove that line and like everything gets kinda weird
